x/auth/client/cli/tx_multisign.go

```diff
@@ -84,13 +84,7 @@ func makeMultiSignCmd(cdc *codec.Codec) func(cmd *cobra.Command, args []string)
 		txBldr := types.NewTxBuilderFromCLI()
 
 		if !viper.GetBool(flagOffline) {
-			addr := multisigInfo.GetAddress()
-			accnum, err := cliCtx.GetAccountNumber(addr)
-			if err != nil {
-				return err
-			}
-
-			seq, err := cliCtx.GetAccountSequence(addr)
+			accnum, seq, err := types.NewAccountRetriever(cliCtx).GetAccountNumberSequence(multisigInfo.GetAddress())
```

Comment:
this line is too long

Reply:
We have no guidelines for maximum line length. Let's agree on a max length and stick to it. Til then, I wouldn't bother too much about it